knative-operator/pkg/controller/knativeserving/kourier/kourier.go

```diff
@@ -38,8 +38,13 @@ func Apply(instance *servingv1alpha1.KnativeServing, api client.Client, scheme *
 		return err
 	}
 	if err := checkDeployments(&manifest, instance, api); err != nil {
-		instance.Status.MarkDependencyInstalling("Kourier")
 		log.Error(err, "")
+		if !instance.Status.IsFullySupported() {
```

Comment:
Isn't this always true from the get-go? Are we still setting the Installing state at all now?

Reply:
See my comment above, I screwed this up.